### PR TITLE
Serialize Contact.fields in test_db

### PR DESCRIPTION
**Pull request by dodobas**

### CODE REVIEW

**Comment by nicpottier** [comment]
Did you track down what was happening with datetimes?

File: temba/utils/management/commands/test_db.py [hunk 1]
```diff
@@ -455,6 +455,42 @@ def create_contacts(self, orgs, locations, num_contacts):
...
+                            'text': six.text_type(c['gender'])
+                        }
+                    if c['age'] is not None:
+                        c['fields_as_json'][six.text_type(org.cache['fields']['age'].uuid)] = {
+                            'text': six.text_type(c['age']),
+                            'decimal': six.text_type(c['age'])
+                        }
+                    if c['joined'] is not None:
+                        c['fields_as_json'][six.text_type(org.cache['fields']['joined'].uuid)] = {
+                            'text': c['joined'].strftime('%Y-%m-%dT%H:%M:%S.%fZ'),
```
Comment (rowanseymour) [inline temba/utils/management/commands/test_db.py]: `c['joined'].isoformat()`  ?

Comment (dodobas) [reply temba/utils/management/commands/test_db.py]: that's the format of the string value that is saved in the database ... returned by `datetime_to_str`

**Comment by dodobas** [comment]
yes, it was a problem with serializing data in `test_db`

File: temba/utils/management/commands/test_db.py [hunk 2]
```diff
@@ -455,6 +455,42 @@ def create_contacts(self, orgs, locations, num_contacts):
...
+                            'text': six.text_type(c['gender'])
+                        }
+                    if c['age'] is not None:
+                        c['fields_as_json'][six.text_type(org.cache['fields']['age'].uuid)] = {
+                            'text': six.text_type(c['age']),
+                            'decimal': six.text_type(c['age'])
+                        }
+                    if c['joined'] is not None:
+                        c['fields_as_json'][six.text_type(org.cache['fields']['joined'].uuid)] = {
+                            'text': datetime_to_str(c['joined']),
```
Comment (rowanseymour) [inline temba/utils/management/commands/test_db.py]: @nicpottier any reason we're not using .isoformat() for all the contact field values that are dates?

Comment (nicpottier) [reply temba/utils/management/commands/test_db.py]: We use `.isoformat()` everywhere. Look in `Contact.set_field` or the migration.

Comment (nicpottier) [reply temba/utils/management/commands/test_db.py]: Note that isoformat is for the `datetime` component, this is the `text` component, which is whatever the user entered.

Comment (rowanseymour) [reply temba/utils/management/commands/test_db.py]: Ah right, was confused by https://github.com/nyaruka/rapidpro/pull/1777#discussion_r176325444

@dodobas let's make this `.isoformat()`

Comment (rowanseymour) [reply temba/utils/management/commands/test_db.py]: Ah ok so for realisim's sake this should be formatted like a human entered date - something like the org format

**Pull request merged**